tests/test_unit/test_auth/test_azure_token_provider.py

```diff
@@ -0,0 +1,8 @@
+from odp_sdk.auth import AzureTokenProvider
+
+
+def test_get_token(azure_token_provider: AzureTokenProvider):
```

Comment:
What does this test actually cover if you are using a mock token provider?

Reply:
Since `AzureTokenProvider` extends `JwtTokenProvider` and `JwtTokenProvider` has been tested in its own tests I am just testing that `authenticate` method implementation in `AzureTokenProvider` returns something when the endpoint is called and returns a non error result. I agree it is not a very complex test but nothing else came to my mind apart from duplicating the `JwtTokenProvider` tests. I would be more than happy to implement any suggestions though.

Reply:
No worries, this should be fine.